arch/xtensa/src/esp32s2/Kconfig

```diff
@@ -287,8 +287,6 @@ config ESP32S2_SPI3
 config ESP32S2_SPIRAM
 	bool "SPI RAM Support"
 	default n
-	select ARCH_HAVE_HEAP2
-	select XTENSA_IMEM_USE_SEPARATE_HEAP
 
 if ESP32S2_SPIRAM && SMP
```

Comment:
This option will never be satisfied, ESP32-S2 is a single core chip, thus does not support SMP.